modules/ROOT/pages/osv2-faq.adoc

```diff
@@ -19,13 +19,19 @@ Mule 3.8.5 and above, and all 4.x versions.
 
 ====
```

Comment:
Are we going to eventually move FAQ text to specific topics?
We want to avoid FAQs, per recent Edit Tip.